projects/1.20/assets/maturi-delight/maturidelight/lang/zh_cn.json

```
@@ -0,0 +1,21 @@
+{
+    "item.maturidelight.suspicious_item": "这个章鱼烧里貌似加了什么东西……",
+    "item.maturidelight.takoyaki_raw": "生章鱼烧",
+    "item.maturidelight.takoyaki": "章鱼烧",
+    "item.maturidelight.takoyaki_soysauce": "淋有酱油的章鱼烧",
+    "item.maturidelight.okonomiyaki_raw": "生什锦烧",
+    "item.maturidelight.okonomiyaki": "什锦烧",
+    "item.maturidelight.okonomiyaki_soysauce": "淋有酱油的什锦烧",
+    "item.maturidelight.dough_raw": "生面团",
+    "block.maturidelight.takoyaki_maker": "章鱼烧器",
+    "block.maturidelight.plate": "铁板",
+    "itemGroup.maturidelight": "祭祀乐事",
+    "advancements.maturidelight.root.title": "祭祀乐事",
+    "advancements.maturidelight.root.description": "认识祭祀活动的食物！",
+    "advancements.maturidelight.making_food.title": "制作美食",
+    "advancements.maturidelight.making_food.description": "用铁板和章鱼烧器做些什么",
+    "advancements.maturidelight.extra_topping.title": "添加调料",
+    "advancements.maturidelight.extra_topping.description": "将酱油淋在章鱼烧或什锦烧上",
+    "advancements.maturidelight.you_got_poison_me.title": "从某种意义上说，中了大奖……",
```

```suggestion
    "advancements.maturidelight.you_got_poison_me.title": "也算是中大奖了……",
```
日本语翻译腔，不要比较好，是这么认为的捏

---

大概率是我这么说话习惯了，原来我也有翻译腔吗（